containerm/cli/cli_command_ctrs_list.go

```diff
@@ -57,6 +61,20 @@ func (cc *listCmd) run(args []string) error {
 	if err != nil {
 		return err
 	}
+	if len(cc.config.filter) > 0 && len(ctrs) > 0 {
+		filtered, err := filterBy(cc.config.filter, ctrs)
+		if err != nil {
+			return err
+		}
+		ctrs = filtered
+	}
+	if cc.config.quiet != false && len(ctrs) > 0 {
+		for _, ctr := range ctrs {
+			fmt.Printf("%s ", ctr.ID)
+		}
+		fmt.Println()
+		return nil
+	}
```

Comment:
Executing `kanto-cm remove $(kanto-cm list --quiet)`, when there are no containers, results in:
```
Error: containers couldn't be removed due to the following reasons: 

* The requested container with ID = No was not found.
* The requested container with ID = found was not found.
* The requested container with ID = containers. was not found.
```

As quiet listing is supposed to work together with other kanto-cm commands, the preferred way is to have no output  and leave to the other command to pop up with an error - e.g. that no container id is provided.

```suggestion
	if cc.config.quiet {
		for i, ctr := range ctrs {
			if i != len(ctrs) - 1 {
				fmt.Printf("%s ", ctr.ID)
			} else {
				fmt.Println(ctr.ID)
			}
		}
		return nil
	}
```